### PR TITLE
VIVI-8270 add px as a termux package

### PR DESCRIPTION
**Pull request by mattd233**

### CODE REVIEW

File: px/main.py
Comment (tristanpenman) [inline px/main.py]: To stick with the styling of commenting out code we don't want, can you comment out everything here up to the original `sys.exit()`?

File: px/main.py
Comment (tristanpenman) [inline px/main.py]: Same here

File: px/main.py
Comment (tristanpenman) [inline px/main.py]: In this case, we should probably comment out the origianl call to `attach_console()`

File: px/wproxy.py
Comment (tristanpenman) [inline px/wproxy.py]: Could actually comment this too

File: px/main.py
Comment (tristanpenman) [inline px/main.py]: Can you remove this line, and uncomment the original `sys.exit()`?

File: px/main.py
Comment (tristanpenman) [inline px/main.py]: Same with this one

File: px/wproxy.py
Comment (tristanpenman) [inline px/wproxy.py]: Only minor, but I think these comments should start one indentation level to the left

File: px/main.py
Comment (tristanpenman) [inline px/main.py]: Leftover line

File: px/main.py
Comment (tristanpenman) [inline px/main.py]: Could fix the comment alignment here, if we're being pedantic

**Review by tristanpenman — APPROVED** [review]
Some minor suggestions, but looks good otherwise. Happy to merge and work from there

**Comment by mattd233** [comment]
Waiting for junmin's branch to get approved first before I merge since this is based on that branch.

**Review by tristanpenman — APPROVED** [review]

**Pull request merged**
